lib/v4/migration-helpers/update-api-folder-structure/index.js

```diff
@@ -6,27 +6,39 @@ const { resolve, join, basename } = require('path');
 const fs = require('fs-extra');
 const _ = require('lodash');
 const chalk = require('chalk');
+const { Liquid } = require('liquidjs');
 
 const pluralize = require('pluralize');
 const runJscodeshift = require('../../utils/run-jscodeshift');
 const { logger } = require('../../../global/utils');
 const updateContentTypes = require('../convert-models-to-content-types');
-const updateRoutes = require('../update-routes');
+const updateConfig = require('../update-application-config');
+const updateRoutes = require('../update-application-routes');
+const updateControllers = require('../update-application-controllers');
+const updateServices = require('../update-application-services');
 const updatePolicies = require('../update-api-policies');
 const renameApiFilesToSingular = require('../rename-api-files-to-singular');
 const { getDirsAtPath, cleanEmptyDirectories } = require('./utils');
 
+const liquidEngine = new Liquid({
+  root: resolve(__dirname, 'templates'),
+  extname: '.liquid'
+})
+
 const updateApiFolderStructure = async (appPath) => {
   const strapiAppPath = resolve(appPath);
   const apiDirCopyPath = join(strapiAppPath, 'src', 'api');
+  const configPath = join(strapiAppPath, 'config');
 
   try {
-    // Copy the folder to a v3 folder for safe keeping
+    // Copy the config folder to the v3 folder for safe keeping
+    await fs.move(configPath, join(strapiAppPath, 'v3', 'config'));
```

Comment:
I'm not sure this should be done here since it isn't related to the api.

I don't think it should be moved because it will still be at the root of the application, just copied to v3 for safe keeping

Could we do this in `update-application-folder-structure`, you probably could create the folder there if it doesn't exist instead of in `update-application-config.js`

Something like:

```js
    const configPath = join(strapiAppPath, 'config')
    const exists = await fs.exists(configPath);
    if (!exists) {
      await fs.copy(configPath, join('v3', 'config'));
    } else {
      // create config path
      await fs.mkdir(configPath);
    }
```